hbook/h1d.go

```diff
@@ -728,6 +728,18 @@ scanLoop:
 	return err
 }
 
+// Counts return a slice of Count, ignoring outerflow.
+func (h *H1D) Counts() []Count {
+	cs := make([]Count, h.Len())
+	for i, bin := range h.Binning.Bins {
+		cs[i].XRange = bin.Range
+		cs[i].Val = bin.Dist.Dist.SumW
+		cs[i].Err.Low = math.Sqrt(bin.Dist.Dist.SumW2) * 0.5
```

Comment:
the way errors are filled for that slice of counts should probably documented in the `Counts()` method.
also: perhaps just use `Dist0D.errW()` ?

Reply:
I have actually went for `h.Value(i)` and `h.Error(i)` - let me know if you prefer `Dist0`-based calls.